std_msgs/Header header
Robot left_robot
Robot right_robot